src/main/java/de/florianmichael/viafabricplus/definition/authlib/KeyPairResponse1_19_0.java

```diff
@@ -0,0 +1,14 @@
+package de.florianmichael.viafabricplus.definition.authlib;
+import com.mojang.authlib.yggdrasil.response.KeyPairResponse;
+import java.nio.ByteBuffer;
+/*
+This library is part of the AuthLib, we are overwriting this class to add a new field.
+ */
+
+public record KeyPairResponse1_19_0() {}
```

Comment:
Delete this class and revert the whole key signing fix like it was in VFP 1.20.1